src/core/service/token/authutils.go

```diff
@@ -100,7 +106,7 @@ func filterAccess(access []*token.ResourceActions, ctx security.Context,
 }
 
 // MakeToken makes a valid jwt token based on parms.
-func MakeToken(username, service string, access []*token.ResourceActions) (*models.Token, error) {
+func MakeToken(username, service string, access []*token.ResourceActions, policyCheck bool) (*models.Token, error) {
```

Comment:
I think @steven-zon mentioned the scan is just a reource action?

Reply:
+ @steven-zou, please add comments

Reply:
updated